CMakeLists.txt

```diff
@@ -188,12 +204,12 @@ if (NOT OPENSSL_OFF)
 	
 	else()
 		include(LibreSSL)
-		use_libre_ssl(${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR})
+		use_libre_ssl("${CMAKE_CURRENT_SOURCE_DIR}" "${CMAKE_CURRENT_BINARY_DIR}")
 		list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/ssl")
 		find_package (OpenSSL REQUIRED)
 	endif()
 if (OPENSSL_FOUND)
-  include_directories(${OPENSSL_INCLUDE_DIR})
+  include_directories("${OPENSSL_INCLUDE_DIR}")
```

Comment:
shouldn't be necessary but this seemed to solve some issues on windows with spaces in build dirs. I soon realized that this caused a host of issues across cmake so this may/should be reverted.